### ISSUE TITLE
Export DOMElement type

### ISSUE THREAD

**Issue by progfay** [vadimdemedes/ink#350]
I want to use [`DOMElement`](https://github.com/vadimdemedes/ink/blob/master/src/dom.ts#L27-L45) type, but it seem not to be exported.

**Usecase: [`measureElement`](https://github.com/vadimdemedes/ink#measureelementref)**

```tsx
import React from 'react'
import { Box, Text, measureElement } from 'ink'

// ↓ Maybe it's not exported
import type { DOMElement } from 'ink'

const Sample: React.FC = () => {
  const ref = React.useRef<DOMElement>(null)

  React.useEffect(() => {
    const {width, height} = measureElement(ref.current)
  })

  return (
    <Box ref={ref}>
      <Text> This is Sample </Text>
    </Box>
  )
}
```

**Comment by progfay** [comment]
Dirty hack to use `DOMElement` type, 

```ts
import type { DOMElement } from 'ink/build/dom'
```

**Comment by vadimdemedes** [comment]
Makes sense. Released in [3.0.2](https://github.com/vadimdemedes/ink/releases/tag/v3.0.2).
